sentences/en/_common.yaml

```diff
@@ -124,7 +124,7 @@ lists:
         out: window
   lock_states:
     values:
-      - in: "[securely ]locked"
+      - in: "[securely] locked"
```

Comment:
This seems to revert changes done in recent history?

The space is part of the optional thing here

Reply:
It was (in the interest of optimization), but we made a decision to undo that change and to standardize the usage of spaces outside optional parts of sentences with the benefit of legibility and lower barriers for contribution.

Reply:
Thanks for the context 👍